### DISCUSSION TITLE
Uncaught TypeError: path__WEBPACK_IMPORTED_MODULE_0___default(...).resolve is not a function

### DISCUSSION THREAD

**Discussion by ylahav** [payloadcms/payload#2275] [Q&A]
# Bug Report

Getting this error when trying to got to url http://<domain-name>/admin
and getting a white board

## Steps to Reproduce

1. I am using local file system ('fs')
2. have the follwing in package.json to wllow working with 'fs'
"browser": {
    "fs": false,
    "os": false,
    "path": false
  }

## Other Details

Payload version - latest

**Comment by jmikrut** [comment]
Hey @ylahav — this is likely something to do with your implementation, and I'm not sure how the `http:///admin` URL would ever work in the first place, but if you would like to provide more information, we're happy to help.

One note though - our Issues are reserved for Payload bugs and issues. But we're happy to help you out on GitHub Discussions and Discord! I'll convert this over to a discussion and we can keep this going there.

**Reply by ylahav** [reply]
The message was incorrect... the correct URL is:
http://localhost:3000/admin...
and the issue is when calling the function path.resolve...
I create a workaround by adding an attribute 'ROOT_DIR' in my env file and it works without any problem.
